### ISSUE TITLE
Update JD calculation in Pulsar Beamformer to use existing function 

### ISSUE THREAD

**Issue by andrerenard** [kotekan/kotekan#365]
See @jrs65 's notes in #363 

We should replace the complex JD calculation with the `ts2mjd` function in `lib/utils/pulsarTiming.hpp`.  And possibly move that function out to a standard utils file, since it seems generally useful.

**Comment by jrs65** [comment]
Agreed. We are building up a large set of timing related tools that I think we should move into a separate time utils file. Maybe `utils/time.hpp`?

At the moment we are trying to pin down some ambiguity regarding leap seconds though so we might want to hold off a little bit.
